frontend/src/styles/GlobalStyles.ts

```diff
@@ -33,16 +33,30 @@ export const GlobalStyles = createGlobalStyle`
     src: url('/fonts/Mont-Regular.otf') format('opentype');
   }
 
-  html {
-    scroll-behavior: smooth;
+  html, body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
     font-family: Mont;
+    background-color: var(--rstyle1-gray-background);
+    display: flex;
+    flex-direction: column;
+    scroll-behavior: smooth;
   }
 
   body {
-    background-color: var(--rstyle1-gray-background);
-    margin: 0;
-    padding: 0; 
-    font-family: Mont;
+    display: flex;
+    flex-direction: column;
+  }
+
+  #App {
```

Comment:
We have many display flex to center, this code can be more simpler and doesn't need to involve so much CSS logic. But ok, if works